packages/common/src/services/audius-backend/solana.ts

```diff
@@ -548,6 +546,7 @@ export const relayVersionedTransaction = async (
     skipPreflight?: boolean
   }
 ) => {
+  const PLACEHOLDER_SIGNATURE = Buffer.from(new Array(64).fill(0))
```

Comment:
Great context! I need to avoid calling Buffer top level because Buffer is undefined when the file is imported. We are doing another Buffer.from in a map a few lines below in that function: https://github.com/AudiusProject/audius-protocol/blob/main/packages/common/src/services/audius-backend/solana.ts#L559

Maybe it's the array fill that's slow on mobile? I can def move that to the top of the file and still generate the buffer in the function